### PR TITLE
Add initial UI structure, dependencies

### PR DESCRIPTION
**Pull request by greenca6**
- Added [reactstrap](https://reactstrap.github.io/) to dependencies
- Added files/directories for the structure of the UI app

Will merge this into the `develop` branch once everything is passing.

Changes here are nowhere near final, and will be updated later on in a future branch/PR.

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -4,9 +4,11 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
+    "bootstrap": "^4.1.3",
     "react": "^16.5.1",
     "react-dom": "^16.5.1",
     "react-scripts": "1.1.5",
+    "reactstrap": "^6.5.0",
```
Comment (greenca6) [inline ui/package.json:14]: The library with a bunch of cool components for us to use. https://reactstrap.github.io/

File: ui/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                  >
+                    Send!
+                  </button>
+                </div>
+              </div>
+              The server says: {serverMessage}
+            </div>
+          </div>
+        </main>
+      </div>
```
Comment (greenca6) [inline ui/src/App.jsx]: Cleaned up some of the Markup here. This will definitely change, just making things slightly prettier for the time being.

**Comment by Sweethezus** [comment]
What's up with Travis failing the pull request?

**Comment by greenca6** [comment]
The pull request is building the target branch, which is `develop`...which I think in this case is okay if that fails since it's a development branch. As to _why_ it's failing, not sure.

**Comment by Sweethezus** [comment]
I think we should wait to figure out _why_ it is failing prior to committing, even though it is a development branch.

**Pull request merged**